fluence.yaml

```diff
@@ -29,11 +29,12 @@
 # appTSPath: ./path/to/app-ts/dir # Optional. If you want to generate ts files including app.ts to be able to access deployed app data in aqua when using FluenceJS,
 # appJSPath: ./path/to/app-js/dir # Optional. If you want to generate js files including app.js to be able to access deployed app data in aqua when using FluenceJS
 
-version: 2
+version: 3
 
 dependencies:
   npm:
     "@fluencelabs/cli": 0.5.4
```

Comment:
why do we have it here still?

Reply:
You mean .yaml? I think I can remove it and put in gitignore